src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -320,7 +320,7 @@ private SourceRepository GetSource(string source)
 
                     foreach (var resource in metadataResources)
                     {
-                        tasks.Add(resource.GetVersions(dep.Id, token));
+                        tasks.Add(resource.GetVersions(dep.Id, Logging.NullLogger.Instance, token));
```

Comment:
The VS API logs to the VS status bar, I'm not sure that callers would want some of the errors messages displayed since they might be expected and handled by the caller.